server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/nano/test/TestNodeWallet.java

```diff
@@ -0,0 +1,44 @@
+package com.generalbytes.batm.server.extensions.extra.nano.test;
+
+import com.generalbytes.batm.common.currencies.CryptoCurrency;
+import com.generalbytes.batm.server.extensions.extra.nano.util.NanoUtil;
+import com.generalbytes.batm.server.extensions.extra.nano.NanoExtensionContext;
+import com.generalbytes.batm.server.extensions.extra.nano.rpc.NanoRpcClient;
+import com.generalbytes.batm.server.extensions.extra.nano.wallet.node.NanoNodeWallet;
+import com.generalbytes.batm.server.extensions.payment.ReceivedAmount;
+
+import java.math.BigDecimal;
+import java.net.URL;
+import java.util.UUID;
+
+/**
+ * THIS CLASS MAY BE IGNORED.
+ * Its only purpose is to test and help during development.
+ */
```

Comment:
could you please remove these classes or move them to `/src/test/` as `@Ignore`-d tests for example?